CNC/Channel-v1.json

```diff
@@ -378,6 +378,46 @@
         }
       ]
     },
+    "Act_Program_Cycle_Time": {
```

Comment:
`Active` would be better than `Act`. And is it worth pushing these two down into a `CNC_Program` schema?

Reply:
Using the original notation. Could start a `v2`?